### PR TITLE
[kube-proxy] fix cve and build for cse edition

### PR DESCRIPTION
**Pull request by borg-z**
## Description

kubeProxy initContainer
kubeProxy kubeProxy127

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: <kebab-case of a module name> | <1st level dir in the repo>
type: fix | feature | chore
summary: <ONE-LINE of what effectively changes for a user>
impact: <what to expect for users, possibly MULTI-LINE>, required if impact_level is high ↓
impact_level: default | high | low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: modules/036-kube-proxy/images/init-container/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+artifact: {{ .ModuleName }}/{{ $.ImageName }}-artifact
+from: {{ .Images.BASE_GOLANG_23_ALPINE }}
+import:
+  - artifact: {{ .ModuleName }}/{{ $.ImageName }}-src-artifact
+    add: /src
+    to: /src
+    before: install
+shell:
+  beforeInstall:
+    {{- include "alpine packages proxy" . | nindent 2 }}
```
Comment (RomanenkoDenys) [inline modules/036-kube-proxy/images/init-container/werf.inc.yaml:20]:
```suggestion
```

File: modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {{- $binaries := "/sbin/xtables-nft-multi /sbin/ip6tables-nft /sbin/ip6tables-nft-restore /sbin/ip6tables-nft-save /sbin/iptables-nft /sbin/iptables-nft-restore /sbin/iptables-nft-save /sbin/xtables-legacy-multi /sbin/iptables /sbin/iptables-restore /sbin/iptables-save /sbin/iptables-legacy /sbin/iptables-legacy-restore /sbin/iptables-legacy-save /sbin/ip6tables /sbin/ip6tables-restore /sbin/ip6tables-save /sbin/ip6tables-legacy /sbin/ip6tables-legacy-restore /sbin/ip6tables-legacy-save /usr/sbin/conntrack /usr/lib64/libnetfilter_conntrack.so*" }}
+
+---
 artifact: {{ .ModuleName }}/{{ $.ImageName }}-artifact
 from: {{ $.Images.BASE_ALT_DEV }}
```
Comment (RomanenkoDenys) [inline modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml:5]: use BASE_ALT_P11 here

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**